spec/hyacinth/adapters/external_identifier_adapter/datacite_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite do
```

Comment:
Missing tests for #update and #tombstone, which are both referenced in DigitalObjectConcerns::PublishBehavior.

Reply:
Test added for #update. #tombstone to be implemented in HYACINTH-876